hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/utils/Pipelines.java

```diff
@@ -482,7 +486,8 @@ public static String opName(String operatorN, Configuration conf) {
   }
 
   public static String opUID(String operatorN, Configuration conf) {
-    return "uid_" + operatorN + "_" + getTablePath(conf);
+    Integer operatorCount = OPERATOR_COUNTERS.merge(operatorN, 1, (oldValue, value) -> oldValue + value);
+    return "uid_" + operatorN + (operatorCount == 1 ? "" : "_" + (operatorCount - 1)) + "_" + getTablePath(conf);
```

Comment:
If we start from 0, there is no need to minus 1 right?

Reply:
It seems like starting from 1.

Reply:
You mean the start number must be 1?

Reply:
If you start from 0, there is no need to do `operatorCount - 1`, the logic does not change.